subprojects/core/src/integTest/groovy/org/gradle/api/tasks/bundling/ArchiveIntegrationTest.groovy

```diff
@@ -944,11 +942,6 @@ class ArchiveIntegrationTest extends AbstractIntegrationSpec {
         "Zip"    | "version"  | "build/distributions/archive.zip"
         "Jar"    | "version"  | "build/libs/archive.jar"
         "Tar"    | "version"  | "build/distributions/archive.tar"
-
```

Comment:
This whole test doesn't make sense anymore because the "version" here was referring to the archive task's version property.  Now that it's been removed, we're setting the _project's_ version.  I think we can delete this entire method.